⬆️(richie) upgrade to 1.0.0-beta.1

jmaupetit opened the pull request:
## Purpose

We are about to deploy an official instance of Richie, thus we need to upgrade its default release in Arnold.

## Proposal

- [x] upgrade Richie to `1.0.0-beta.1`
- [x] fix jobs command (since the project's tree evolved)

sampaccoud commented on apps/marsha/templates/app/job_collectstatic.yml.j2:
```diff
@@ -41,7 +41,11 @@ spec:
         # We point to a local registry image build for this "live" image (see
         # ImageStream and BuildConfig templates)
         image: "{{ internal_docker_registry }}/{{ project_name }}/marsha:{{ marsha_image_tag }}-live"
-        command: ["python", "manage.py", "collectstatic", "--noinput"]
```
why not just:
```suggestion
command: ["python", "sandbox/manage.py", "collectstatic", "--noinput"]
```

jmaupetit replied:
This is easier to avoid having to modify the PYTHONPATH. But I made a mistake by modifying Masha's jobs instead of Richie's ones. :sweat_smile:

jmaupetit committed:
⬆️(richie) upgrade to 1.0.0-beta.1

Upgrade Richie to its latest release. As the project tree evolved, we
had to fix jobs command and update the "regenerate indexes" command that
has been renamed "bootstrap elasticsearch".

This later one requires the database to have been set, hence, we renamed
job template files to fix execution ordering as:

1. database migrations
2. collect statics
3. ES bootstrap

lunika reviewed (approved):
![](https://media.giphy.com/media/YKdm91XONLTaw/giphy-downsized.gif)
